apps/docs/src/guide/transactions/transaction-response.md

```diff
@@ -0,0 +1,23 @@
+# Transaction Response
+
+Once a transaction has been submitted, you may want to extract information regarding the result of the transaction. The SDK offers a `TransactionResponse` class with helper methods to expose the following information:
+
+- The transaction ID
+- The status (submitted, success, squeezed out, or failure)
+- Receipts (return data, logs, mints/burns, transfers and panic/reverts)
+- Gas fees and usages
+- The raw payload of the transaction including inputs and outputs
```

Comment:
This one would be removed in #1644 so I'm not sure if we should be mentioning it here at all. I'd really like people to _not_ know about it so that they don't break their code when it's removed.

Reply:
By one, you mean just `reciepts`?

Reply:
- The raw payload of the transaction including inputs and outputs

Reply:
The whole raw payload

Reply:
Based on #1644 you're right this does make the response redundant for a tx just submitted. But we will still want this to fetch info about a tx that was submitted historically?

Reply:
Going to resolve this conversation and leave this to be unimplemented by #1644, looks like there are blockers up the chain.